test/functions/lydia/userDefinedDefaultFunction/redefine-int.chpl

```diff
@@ -1,4 +1,4 @@
-inline proc _defaultOf(type t) param where (isIntegralType(t)) return 5:t;
+inline proc _defaultOf(type t:int) param return 5:t;
```

Comment:
`t:integral` ?

Reply:
These tests check that a user-defined _defaultOf can work in various situations. I left them using the specific type to reduce the possible conflicts with other shuffling of _defaultOf in the modules. Either way, I don't view _defaultOf as a user-facing feature so my personal view is that all these tests defining a _defaultOf should just be removed.

Reply:
That's fine, then.